src/mcts/search.cc

```diff
@@ -530,24 +533,32 @@ bool SearchWorker::IsSearchActive() const {
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 void SearchWorker::InitializeIteration(
     std::unique_ptr<NetworkComputation> computation) {
-  nodes_to_process_.clear();
   computation_ = std::make_unique<CachingComputation>(std::move(computation),
                                                       search_->cache_);
+  minibatch_.clear();
 }
 
 // 2. Gather minibatch.
 // ~~~~~~~~~~~~~~~~~~~~
 void SearchWorker::GatherMinibatch() {
-  int nodes_found = 0;
+  // Total number of nodes to process.
+  int minibatch_size = 0;
   int collisions_found = 0;
+  // Number of nodes processed out of order.
+  int number_out_of_order = 0;
 
   // Gather nodes to process in the current batch.
-  while (nodes_found < search_->kMiniBatchSize) {
+  // If we had too many (kMiniBatchSize) nodes out of order, also exit so that
+  // search has a chance to exit search.
```

Comment:
I think that there's a redundant word in this comment?

Reply:
Yeah it was weirdly stated, probably that way is better.